.github/workflows/pr.yml

```diff
@@ -397,9 +397,40 @@ jobs:
           python-version: ${{matrix.python}}
         if: matrix.test-suite == 'integration' || matrix.test-suite == 'notebook'
 
+      - uses: conda-incubator/setup-miniconda@v2
+        if: matrix.test-suite == 'conda'
+        with:
+          auto-update-conda: false
+          python-version: ${{ matrix.python }}
+          miniconda-version: "latest"
+          activate-environment: foo
+
+      - name: Conda info
+        if: matrix.test-suite == 'conda'
+        # Use `bash -l {0}` when using the conda actions.
+        shell: bash -l {0}
+        run: |
+          conda info
+          conda list
+
+      - name: Python Info
+        # Use `bash -l {0}` when using the conda actions.
+        shell: bash -l {0}
+        run: |
+          python --version
+          python -c 'import sys;print(sys.executable)'
```

Comment:
Useful for debugging (see what python env was used)
We found we weren't using the right one & Rich fixed issues in another PR.

Reply:
Also need to use `bash -l {0}` to ensure we use the right python env (when using conda).